src/main/resources/templates/db_scripts/setNotNullConstraintForJsonb.sql

```diff
@@ -0,0 +1,11 @@
+DO $$ BEGIN ALTER TABLE actual_opening_hours ALTER COLUMN jsonb SET NOT NULL; EXCEPTION WHEN OTHERS THEN END; $$;
+DO $$ BEGIN ALTER TABLE exceptional_hours ALTER COLUMN jsonb SET NOT NULL; EXCEPTION WHEN OTHERS THEN END; $$;
+DO $$ BEGIN ALTER TABLE openings ALTER COLUMN jsonb SET NOT NULL; EXCEPTION WHEN OTHERS THEN END; $$;
+DO $$ BEGIN ALTER TABLE regular_hours ALTER COLUMN jsonb SET NOT NULL; EXCEPTION WHEN OTHERS THEN END; $$;
+DO $$ BEGIN ALTER TABLE exceptions ALTER COLUMN jsonb SET NOT NULL; EXCEPTION WHEN OTHERS THEN END; $$;
```

Comment:
The `creation_date` needs to get converted from `timestamp with time zone` to `timestamp without time zone`.

Reply:
updated

Reply:
Thanks, resolved.